app/components/section_item/index.tsx

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback, useMemo} from 'react';
+import {
+    Switch,
+    Text,
+    TouchableOpacity,
+    View,
+} from 'react-native';
+
+import CompassIcon from '@components/compass_icon';
+import {useTheme} from '@context/theme';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+import {typography} from '@utils/typography';
+
+const ActionTypes = {
+    ARROW: 'arrow',
+    DEFAULT: 'default',
+    TOGGLE: 'toggle',
+    SELECT: 'select',
+};
+
+const getStyleSheet = makeStyleSheetFromTheme((theme) => {
+    return {
+        container: {
+            flexDirection: 'row',
+            alignItems: 'center',
+        },
+        singleContainer: {
+            alignItems: 'center',
+            flex: 1,
+            flexDirection: 'row',
+            height: 45,
+        },
+        doubleContainer: {
+            flex: 1,
+            flexDirection: 'column',
+            height: 69,
+            justifyContent: 'center',
+        },
+        label: {
+            color: theme.centerChannelColor,
+            ...typography('Body', 600, 'SemiBold'),
+            fontSize: 16,
+            lineHeight: 24,
+        },
+        description: {
+            color: changeOpacity(theme.centerChannelColor, 0.6),
+            ...typography('Body', 400, 'Regular'),
+            fontSize: 12,
+            lineHeight: 16,
+            marginTop: 3,
+        },
+        arrow: {
+            color: changeOpacity(theme.centerChannelColor, 0.25),
+            fontSize: 24,
+        },
+        labelContainer: {
+            flex: 0,
+            flexDirection: 'row',
+        },
+    };
+});
+
+type Props = {
+    testID?: string;
```

Comment:
Can `testID` be removed if not used ?  If not, then can we assign a default value to the `testID` for if we leave it as `undefined`, then `undefined` will be concatenated to each string in the code below?